pkg/allowedpaths/cmd.go

```diff
@@ -0,0 +1,784 @@
+package allowedpaths
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"runtime"
+)
+
+type AllowedCommand func(ctx context.Context, arg ...string) (*exec.Cmd, error)
+
+func Airport(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+	if runtime.GOOS != "darwin" {
+		return nil, errors.New("airport supported on darwin only")
+	}
```

Comment:
This loses the compile time enforcement -- I think we should define this in an `_darwin.go` file. Nothing on another platform should call it.

Reply:
Yeah, agreed -- I'll go through and audit to see what we can move. I know there are a couple places where we call a darwin- or posix-specific func in a file that's still compiled for all platforms, so I think I'll have to do a little bit of rearranging.

Reply:
Not as bad as I thought it'd be -- just a couple checkups and a couple tables to adjust.